src/corner_kick/src/components/LayersPanel/__stories__/index.tsx

```diff
@@ -18,23 +18,23 @@ stories.add('with layers', () => (
         <LayersPanel
             layers={[
                 {
-                    id: 'friendly_robots',
                     name: 'Friendly Robots',
```

Comment:
Are these the layer names from messages? 

If so, I currently have the following layer names defined on VM: "field", "ball", "ball_vel", "friendly_robot", "friendly_robot_vel", "enemy_robot", "enemy_robot_vel", "nav", "rule", "passing", "test", "misc". (util/visualizer_messenger/visualizer_messenger.cpp)

We either needs to have the same naming, or have some 1:1 mapping between layer name from the message, and layer name for UI.

Reply:
This is a fixture I made for my visual tests and doesn't make any assumptions regarding the layers of the AI. I can modify it however to be more in line with the layers in the AI

Reply:
I don't think it's a huge issue here if this is just for the storybook. IIRC, the visualizer can handle whatever layers get passed to it right? It doesn't need to know anything about what layers the AI might send it?

Reply:
Yup this is correct

Reply:
Gotcha! o7